src/fu-engine.c

```diff
@@ -118,6 +118,9 @@ struct _FuEngine {
 	GMainLoop *acquiesce_loop;
 	guint acquiesce_id;
 	guint acquiesce_delay;
+
+	gchar *machine_kind_str_mockup;
+	FuSmbiosChassisKind chassis_type_mockup;
```

Comment:
we shouldn't be storing override variables in `struct _FuEngine` most likely.  I would rather see an override in fu-smbios related to this so that all the other callers get this information correctly too.

Reply:
I can look into fu-smbios first.  machine_kind is placed in fu-daemon. I have to find a way to set it.